app/(main)/league/[leagueId]/entry/[entryId]/week/Week.test.tsx

```diff
@@ -94,36 +78,56 @@ global.ResizeObserver = jest.fn().mockImplementation(() => ({
   disconnect: jest.fn(),
 }));
 
-describe('Week', () => {
-  const teamSelect = 'Browns';
-  const NFLTeams = [{ teamName: 'Browns', teamId: '1234', teamLogo: 'browns' }];
-  const user = { id: '12345', email: 'email@example.com', leagues: [] };
-  const entry = 'mockEntry';
-  const league = 'mockLeague';
-  const week = '1';
-  const setUserPick = jest.fn();
-  const updateWeeklyPicks = jest.fn();
-  const mockGetCurrentLeague = getCurrentLeague as jest.Mock;
-  const mockCreateWeeklyPicks = createWeeklyPicks as jest.Mock;
-
-  const weeklyPicks: IWeeklyPicks = {
-    leagueId: '123',
-    gameWeekId: '123456',
-    userResults: {},
-  };
+// constants
+const teamSelect = 'Browns';
+const NFLTeams = [
+  {
+    teamId: 'browns',
+    teamName: 'Browns',
+    teamLogo: 'https://example.com/browns.png',
+  },
+  {
+    teamId: 'packers',
+    teamName: 'Packers',
+    teamLogo: 'https://example.com/packers.png',
+  },
+];
+const user = {
+  documentId: 'mockDocument',
+  id: '123',
+  email: 'email@example.com',
+  leagues: ['123'],
+};
+const entry = '123';
+const league = '123';
+const week = '1';
+const weeklyPicks: IWeeklyPicks = {
+  leagueId: '123',
+  gameWeekId: '123456',
+  userResults: {},
+};
 
-  const teamID = NFLTeams[0].teamName;
+const teamName = NFLTeams[0].teamName;
 
-  const updatedWeeklyPicks = {
-    ...weeklyPicks.userResults,
-    [user.id]: {
-      ...weeklyPicks.userResults[user.id],
-      [entry]: {
-        ...weeklyPicks.userResults[user.id]?.[entry],
-        ...parseUserPick(user.id, entry, teamID || '')[user.id][entry],
-      },
+const updatedWeeklyPicks = {
+  ...weeklyPicks.userResults,
+  [user.id]: {
+    ...weeklyPicks.userResults[user.id],
+    [entry]: {
+      ...weeklyPicks.userResults[user.id]?.[entry],
+      ...parseUserPick(user.id, entry, teamName || '')[user.id][entry],
     },
-  };
+  },
+};
+
+describe('Week', () => {
```

Comment:
nit: I feel like the describe copy here could be expanded on just a little bit, like `League Week` or `Survivor League Week`

Reply:
✅ updated to "League Week Picks"